src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/RelatedAppointment.kt

```diff
@@ -24,6 +25,7 @@ data class RecurringAppointment(
 
   val repeatPeriod: RepeatPeriod,
   val count: Long,
+  val startTime: LocalDateTime,
```

Comment:
Not sure about this name, but given the context is RecurringAppointment maybe it's enough?